topology/Stanford University/SLAC/WT2.yaml

```diff
@@ -225,4 +226,33 @@ Resources:
           endpoint: psnr-farm10.slac.stanford.edu
     VOOwnership:
       ATLAS: 100
+  SLAC_VARNISH_FRONTIER_NRP:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          ID: 2d51aa6faa109d97dec985456a69cb3dab518c8e
+          Name: Steffen Luitz
+        Secondary:
+          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
+          Name: Wei Yang
+      Security Contact:
+        Primary:
+          ID: 2d51aa6faa109d97dec985456a69cb3dab518c8e
+          Name: Steffen Luitz
+        Secondary:
+          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
+          Name: Wei Yang
```

Comment:
Are Wei and Steffen the folks we should contact for security and administrative issues?

Reply:
only me, but I don't have an ID.

Reply:
Hey @ivukotic in that case you'll need to register to get one https://osg-htc.org/docs/common/contact-registration/

Reply:
I just tried but it does not work... I tried following the instructions and got this:
![image](https://github.com/opensciencegrid/topology/assets/1736984/b36e9762-fa07-4edd-817e-664e0490530e)

Reply:
Ah right, that's because you're special and you've already got an account through the OpenData Facility. I added you to the right group so your ID is `OSG1000064` corresponding to name `Ilija Vukotic`

Reply:
thanks. updated now